### DISCUSSION TITLE
Syntax highlighting: ansi + language

### DISCUSSION THREAD

**Discussion by lazarusA** [vuejs/vitepress#3518] [Q&A]
Currently one can do either:

```julia
julia> printstyled("This should be in bold light cyan.", color=:light_cyan, bold=true)
[96m[1mThis should be in bold light cyan.[22m[39m
```
to get highlighted code for the language functions.

Or `ansi` to get coloured output via:

````shell
```ansi
julia> printstyled("This should be in bold light cyan.", color=:light_cyan, bold=true)
[96m[1mThis should be in bold light cyan.[22m[39m

```
````

but the language-coding colours are gone. Is there a way to combine both? ansi + language?

**Comment by brc-dd** [comment]
Currently no. Please create a feature request at https://github.com/shikijs/shiki instead.
